lib/hare/context/action/qos.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Hare.Context.Action.Qos do
```

Comment:
Although I'm not the author of `hare`, but I think `QoS` over `Qos` is more correct :)